src/script.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# prefixes all lines of commands written to stdout with datetime
+PS4='\000[$(date)]\011'
+export TZ=Europe/London
+set -exo pipefail
+
+# set frequency of instance usage in logs to 10 seconds
+kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)